### ISSUE TITLE
Month name to number

### ISSUE THREAD

**Issue by marijusGood** [uxsolutions/bootstrap-datepicker#2699]
### Expected behaviour
Hi team, I think you should have an option(or make it a default) that months written as words should be converted to numbers, for example, 01-February-2024 should be 01-02-2024.

### Actual behaviour
01-February-2024 defaults to current date

### Datepicker version used

1.9.0

**Comment by ralfhauser** [comment]
Yes the month-as-number and the month-as-name-string modes should work simultaneously.

(and obviously, it should be configurable in which languages the month-as-name-string should be recognized - i.e. which language resources to support for this)
